testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -179,16 +184,16 @@ impl Experiment for PerformanceBenchmark {
         context.report.report_metric(&self, "avg_tps", avg_tps);
         context
             .report
-            .report_metric(&self, "avg_latency", avg_latency);
+            .report_metric(&self, "avg_latency", avg_latency_client as f64);
         info!("avg_txns_per_block: {}", avg_txns_per_block);
         let expired_text = if expired_txn == 0 {
             "no expired txns".to_string()
         } else {
             format!("(!) expired {} out of {} txns", expired_txn, submitted_txn)
         };
         context.report.report_text(format!(
-            "{} : {:.0} TPS, {:.1} ms latency, {}",
-            self, avg_tps, avg_latency, expired_text
+            "{} : {:.0} TPS, {:.1} ms client side latency, {}",
```

Comment:
I think you can remove 'client side' from this message - it's more or less important only once (when this change is landed) after this is just redundant detail repeating every time